static/src/js/components/CustomToggle/MoreActionsToggle.js

```diff
@@ -1,44 +1,39 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
 
 import CustomToggle from './CustomToggle'
 
 import './MoreActionsToggle.scss'
 
-// Need to use a Class component here so this works with the refs passed from
-// the parent react-bootstrap component
-// eslint-disable-next-line react/prefer-stateless-function
-export class ToggleMoreActions extends Component {
-  render() {
-    const {
-      className,
-      onClick
-    } = this.props
-
-    const moreActionsToggleClassNames = classNames(
-      className,
-      'more-actions-toggle'
-    )
-
-    return (
-      <CustomToggle
-        className={moreActionsToggleClassNames}
-        onClick={onClick}
-        title="More actions"
-        icon="ellipsis-v"
-      />
-    )
-  }
-}
-
-ToggleMoreActions.defaultProps = {
+// eslint-disable-next-line react/display-name
```

Comment:
Why is this needed here and in GranuleResultsDataLinksButton.js but not in CustomToggle.js ?